src/test/TestGhoBase.t.sol

```diff
@@ -78,6 +81,18 @@ import {FixedFeeStrategy} from '../contracts/facilitators/gsm/feeStrategy/FixedF
 import {SampleLiquidator} from '../contracts/facilitators/gsm/misc/SampleLiquidator.sol';
 import {SampleSwapFreezer} from '../contracts/facilitators/gsm/misc/SampleSwapFreezer.sol';
 import {GsmRegistry} from '../contracts/facilitators/gsm/misc/GsmRegistry.sol';
+import {IGhoGsmSteward} from '../contracts/misc/interfaces/IGhoGsmSteward.sol';
+import {GhoGsmSteward} from '../contracts/misc/GhoGsmSteward.sol';
+import {FixedFeeStrategyFactory} from 'src/contracts/facilitators/gsm/feeStrategy/FixedFeeStrategyFactory.sol';
+
+// CCIP contracts
+import {UpgradeableTokenPool} from '../contracts/misc/deps/Dependencies.sol';
+import {UpgradeableLockReleaseTokenPool} from '../contracts/misc/deps/Dependencies.sol';
+import {UpgradeableBurnMintTokenPool} from '../contracts/misc/deps/Dependencies.sol';
```

Comment:
I think we should mock CCIP token pools

Reply:
I have a commit where I had that, but since one of the pools is a dependency of GhoCcipSteward, I don't want to import a mock. I need a stripped down version to deploy for the tests. I think to not duplicate code, we can just have the one shortened version we import and use for tests. What do you think?

Reply:
I have a reduced version and a mock now, along with an interface with only needed functions as the GhoCcipSteward.